Manatee.Trello/Contracts/IAttachmentCollection.cs

```diff
@@ -17,13 +17,14 @@ public interface IAttachmentCollection : IReadOnlyCollection<IAttachment>
 		/// <returns>The <see cref="IAttachment"/> generated by Trello.</returns>
 		Task<IAttachment> Add(string url, string name = null, CancellationToken ct = default);
 
-		/// <summary>
-		/// Adds an attachment to a card by uploading data.
-		/// </summary>
-		/// <param name="data">The byte data of the file to attach.</param>
-		/// <param name="name">A name for the attachment.</param>
-		/// <param name="ct">(Optional) A cancellation token for async processing.</param>
-		/// <returns>The <see cref="IAttachment"/> generated by Trello.</returns>
-		Task<IAttachment> Add(byte[] data, string name, CancellationToken ct = default);
-	}
+        /// <summary>
+        /// Adds an attachment to a card by uploading data.
+        /// </summary>
+        /// <param name="filepath">The path of the file to attach.</param>
+        /// <param name="name">A name for the attachment.</param>
+        /// <param name="ct">(Optional) A cancellation token for async processing.</param>
+        /// <returns>The <see cref="IAttachment"/> generated by Trello.</returns>
+        Task<IAttachment> AddAttachment(string filepath, string name, CancellationToken ct = default);
```

Comment:
You're still replacing the old method.  Can you just add an override?

Also, please rename to `AddFile`.